resources/services/security/security_assessments_test.go

```diff
@@ -0,0 +1,44 @@
+package security
```

Comment:
rename file to security_assessments_mock_test

Reply:
Fixed, thanks.